packages/react/src/Link/Link.tsx

```diff
@@ -65,17 +73,22 @@ export function Link({
 
   return (
     <a
-      className={clsx(styles.Link, styles[`Link--${size}`], className)}
+      className={clsx(styles.Link, styles[`Link--${size}`], styles[`Link--arrow-${arrowDirection}`], className)}
       onMouseEnter={handleMouseEnter}
       onMouseLeave={handleMouseLeave}
       onFocus={handleOnFocus}
       onBlur={handleOnBlur}
       {...props}
     >
+      {arrowDirection === 'start' && (
+        <ExpandableArrow className={styles['Link-arrow']} expanded={isHovered || isFocused} reverse />
```

Comment:
Could we add the `hidden` prop to both `ExpandableArrow` SVGs? We'd hide them since they are mostly decorative.

Reply:
Yep, added via the Link prop. Thanks Tyler.